ui-tests/src/test/java/io/syndesis/qe/steps/customizations/extensions/ExtensionSteps.java

```diff
@@ -36,39 +32,35 @@
     private TechExtensionsListComponent techExtensionsListComponent = new TechExtensionsListComponent();
     private ModalDialogPage modalDialogPage = new ModalDialogPage();
 
+    private static final class NavigationElements {
+        public static final String CUSTOMIZATIONS_NAV = "Customizations";
+        public static final String EXTENSION_NAV = "Extensions";
+        public static final String EXTENSION_DETAILS_NAV = "Extensions Details";
+
+        public static final String IMPORT_EXTENSION_BUTTON = "Import Extension";
+    }
+
     @Given("^imported extensions$")
     public void importExtension(DataTable extensionsData) throws Throwable {
         CommonSteps commonSteps = new CommonSteps();
 
-        String extensionsLink = "Extensions";
-        String importButton = "Import Extension";
-
         List<List<String>> dataTable = extensionsData.raw();
 
         for (List<String> dataRow : dataTable) {
-            String extensionName = dataRow.get(0);
-            String extensionFileName = dataRow.get(1);
 
-            commonSteps.navigateTo("", "Customizations");
-            commonSteps.validatePage("", "Customizations");
+            commonSteps.navigateTo("", NavigationElements.CUSTOMIZATIONS_NAV);
+            commonSteps.validatePage("", NavigationElements.CUSTOMIZATIONS_NAV);
 
-            commonSteps.clickOnLink(extensionsLink);
-            commonSteps.validatePage("", "Extensions");
+            commonSteps.clickOnLink(NavigationElements.EXTENSION_NAV);
+            commonSteps.validatePage("", NavigationElements.EXTENSION_NAV);
 
-            SelenideElement techExtensionItem = techExtensionsListComponent.getExtensionItem(extensionName);
+            commonSteps.clickOnButton(NavigationElements.IMPORT_EXTENSION_BUTTON);
+            commonSteps.validatePage("", NavigationElements.IMPORT_EXTENSION_BUTTON);
 
-            if (techExtensionItem != null) {
```

Comment:
Don't we need this if anymore?

Reply:
Well, it would be test logical error as we always start with Clean app state. Currently this can not happen unless you do it on purpose.